pkgs/development/tools/language-servers/neocmakelsp/default.nix

```
@@ -20,8 +20,8 @@ rustPlatform.buildRustPackage rec {
     description = "A cmake lsp based on tower-lsp and treesitter";
     homepage = "https://github.com/Decodetalkers/neocmakelsp";
     license = licenses.mit;
-    platforms = platforms.linux;
-    maintainers = with maintainers; [ rewine ];
+    platforms = platforms.linux ++ [ "x86_64-darwin" "aarch64-darwin" ];
```

```suggestion
    platforms = platforms.unix;
```

Unix includes both linux and darwin.